src/components/Layouts/Primary/TopBar/Items/NavMenu/NavMenu.tsx

```diff
@@ -41,27 +43,72 @@ const useStyles = makeStyles(() => ({
     transform: 'rotate(180deg)',
   },
   subMenu: {
-    backgroundColor: theme.palette.cruGrayDark.main,
+    backgroundImage: `linear-gradient(0deg, ${theme.palette.cruGrayDark.main}, ${theme.palette.cruGrayDark.main})`,
```

Comment:
Can you share some context here? Seems like this is a gradient from `cruGrayDark` to itself... 🤷‍♂️

Reply:
Ok... I didn't notice the `backgroundBlendMode` before. Does that not work with ` backgroundColor: theme.palette.cruGrayDark.main`?

Reply:
I don't think theres a way to blend two different backgroundcolors because one will just overwrite the other, but it does work if you have a background image(solid gradient) and a background color

![image](https://user-images.githubusercontent.com/16872059/134404668-3fdcfb35-adeb-47bf-a1c2-0497cdb54eea.png)

Reply:
Alrighty 👍